src/go/k8s/controllers/redpanda/cluster_controller_scale_test.go

```diff
@@ -39,8 +38,6 @@ var _ = Describe("Redpanda cluster scale resource", func() {
 		intervalShort = time.Millisecond * 20
 	)
 
-	ctrl.SetLogger(logger.NewLogger(logger.Options{}))
```

Comment:
If previous commit added set logger function and this commit removes it maybe remove it from 8912903b commit?